conv_test.go

```diff
@@ -0,0 +1,61 @@
+package golgi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gorgonia.org/gorgonia"
+	"gorgonia.org/tensor"
+)
+
+func TestConvNet(t *testing.T) {
+	c := require.New(t)
+
+	bs := 32
+	convKS := tensor.Shape{3, 3}
+	mpKS := tensor.Shape{2, 2}
+
+	g := gorgonia.NewGraph()
+	x := gorgonia.NewTensor(g, tensor.Float64, 4, gorgonia.WithName("x"), gorgonia.WithShape(bs, 1, 28, 28), gorgonia.WithInit(gorgonia.GlorotU(1)))
+	y := gorgonia.NewMatrix(g, tensor.Float64, gorgonia.WithName("y"), gorgonia.WithShape(bs, 10), gorgonia.WithInit(gorgonia.GlorotU(1)))
+
+	nn, err := ComposeSeq(
+		x,
+		L(ConsConv, WithName("layer 0"), WithSize(bs, 1), WithKernelShape(convKS)),
+		L(ConsMaxPool, WithName("layer 0"), WithKernelShape(mpKS), WithProbability(0.2)),
```

Comment:
I don't understand why `Dropout` cannot be added after this line... something like this:

```
L(ConsMaxPool, WithName("layer 0"), WithKernelShape(mpKS)),
L(ConsDropout, WithProbability(0.2)),`
```

Reply:
it can, it'd be the same. I'll do that